src/core/lib/iomgr/systemd_utils.cc

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
nit: This should be a C++-style comment, each line starting with `//`. Given that we don't have a linter for this yet, I think we can fix it later. See https://github.com/grpc/grpc/pull/31923 for context.